### ISSUE TITLE
logs/logs.txt is resolved using ${PATH}

### ISSUE THREAD

**Issue by HippoBaro** [FocusCompany/daemon#20]
On UNIX systems and Windows, when using `ostream` with relative paths, the final path is resolved by appending the relative target to the current path, i.e the path from which the program was invoked.

In our case, that means that our log sink `logs/logs.txt` changes location based on where the daemon was invoked. And because the software can't create the `logs/` directory itself, it crashes unless launched from the correct location.

This is a big problem when installing the daemon because we really want the logs' location to be consistent.

Logging to a file seems like a good idea but will need more platform-specific work (resolve the binary location etc.)

> The same thing applies to `daemon.config`

**Comment by HippoBaro** [comment]
To be clear, on OSX I have for example:

```
$ ninja install
[...]
-- Install configuration: ""
-- Up-to-date: /usr/local/bin/daemon.config
-- Up-to-date: /usr/local/bin/logs
-- Installing: /usr/local/bin/focus-daemon
-- Installing: /usr/local/bin/printAppTitle.scpt
$ pwd
/Users/hbarraud/Repos/Focus/daemon/build
$ focus-daemon
        *****  Starting Focus daemon on MacOs Platform  *****
[2018-05-02 12:31:55.075] [thread 2912154] [info]               : FocusDaemon is running
[2018-05-02 12:31:55.076] [thread 2912154] [info]               : FocusEventManager is running
[2018-05-02 12:31:55.076] [thread 2912154] [error]              : Missing configuration file
[2018-05-02 12:31:55.076] [thread 2912154] [info]               : Attempt to login
[2018-05-02 12:31:55.076] [thread 2912154] [error]              : Can not establish connection with authentication server
```

The daemon tries to load `daemon.config` but fails because in reality it tries opening `/Users/hbarraud/Repos/Focus/daemon/build/daemon.config` which does not exists.

**Comment by HippoBaro** [comment]
Fixed in #22
